OpenLIFUProtocolConfig/OpenLIFUProtocolConfig.py

```diff
@@ -108,6 +109,67 @@ class OpenLIFUProtocolConfigParameterNode:
 
     """
 
+#
+# OpenLIFUProtocolConfigDialogs
+#
+
+class ProtocolSelectionFromDatabaseDialog(qt.QDialog):
+    """ Create new protocol selection from database dialog """
+
+    def __init__(self, dataLogic, parent=None):
+        super().__init__(slicer.util.mainWindow() if parent == "mainWindow" else parent)
+        """ Args:
+                dataLogic: instance of the data logic, needed to access the loaded database
+        """
+
+        self.setWindowTitle("Select a Protocol")
+        self.setWindowModality(1)
+
+        self.dataLogic = dataLogic
+        self.selected_protocol = None
+
+        self.setup()
+
+    def setup(self):
+
+        self.boxLayout = qt.QVBoxLayout()
+        self.setLayout(self.boxLayout)
+
+        self.listWidget = qt.QListWidget(self)
```

Comment:
It's pretty intuitive to double click an item in this listWidget to try and make and accept that selection in one go. If that's not just me and you agree it's an intuitive thing to do, then I suggest making double click work that way.